src/explore-education-statistics-frontend/.env

```diff
@@ -4,3 +4,4 @@ NOTIFICATION_API_BASE_URL=http://localhost:7073/api
 GA_TRACKING_ID=
 PUBLIC_URL=http://localhost:3000/
 APP_ENV=Local
+HOSTNAME=localhost
```

Comment:
Does this have any impact when deploying to Azure?  Do we need to set HOSTNAME in the real environments?

Reply:
Pretty sure we don't need this as we can use `URL` on the `PUBLIC_URL` variable i.e.

```ts
const url = new URL(process.env.PUBLIC_URL);

url.hostname // outputs localhost if PUBLIC_URL is http://localhost:3000/
```

Reply:
Oh yeah If course it's not needed. Removed now, thanks.